prime-dotnet-webapi/ViewModels/SiteRegistration/OrganizationListViewModel.cs

```diff
@@ -18,6 +18,7 @@ public class OrganizationListViewModel
         public bool Completed { get; set; }
         public bool HasAcceptedAgreement { get; set; }
         public bool HasSubmittedSite { get; set; }
+        public bool IsUnderReview { get; set; }
```

Comment:
While the JIRA ticket says "the organization is under review", keep in mind the text in the ticket is often the product of brainstorming / imprecise language during ticket refinement. In my opinion, the organization is not under review in any way by having a claim on it. Calling it "IsUnderReview" is rather misleading and suggests that Organizations can be approved / have state.  I would prefer if this property was "HasClaim" or something similar. 
HasClaim"